extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -2931,7 +2948,7 @@ static public function createVariable( $cv_name, $cv_variable_type, $cv_access_l
 	static public function changeVariable( $cv_variable_id, $cv_name, $cv_variable_type, $cv_access_level, $cv_variable_group, $cv_description ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$cv_variable_type". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)